tests/e2e/support/objects/app-admin-settings/users/index.ts

```diff
@@ -104,9 +104,19 @@ export class Users {
     action: string
   }): Promise<void> {
     const uuid = this.getUUID({ key })
+
     await openEditPanel({ page: this.#page, uuid, action })
     await changeUser({ uuid, attribute: attribute, value: value, page: this.#page })
+    const currentUser = this.#usersEnvironment.getCreatedUser({ key })
+
+    if (attribute !== 'role') {
```

Comment:
Maybe we can do it as 
```suggestion
    if (attribute === "displayName'') {
```
or name, since we are only changing the name in `updateUserStore` right?